docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
     working_dir: /mermaid
     mem_limit: '2G'
     environment:
-      - NODE_OPTIONS=--max_old_space_size=2048
+      - NODE_OPTIONS=--max_old_space_size=4096
```

Comment:
This seems weird. We're setting a 4GiB heap size limit here, but just above, we have a `mem_limit` of `'2G'`. Is this right? It seems like Docker will just limit us to 2GiB anyway.

Reply:
Fixed

Reply:
Checked one more time. We need to increase it, and I increased mem limit as well